src/frontend/get-jira-configuration.ts

```diff
@@ -50,18 +52,26 @@ const formatDate = function (date) {
 	};
 };
 
-export const getFailedConnections = (installations, subscriptions) => {
-	return installations
+export const getFailedConnections = (
+	installations: FailedInstallations[] | typeof Installation[],
+	subscriptions: SubscriptionClass[]
+) => {
+	return (installations as FailedInstallations[])
```

Comment:
I wouldn't cast it here since right now, they're not that type until they get filtered.